querybook/webapp/components/AIAssistant/QueryCellTitle.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useState } from 'react';
+
+import { IconButton } from 'ui/Button/IconButton';
+import { ResizableTextArea } from 'ui/ResizableTextArea/ResizableTextArea';
+
+import './QueryCellTitle.scss';
+
+export const QueryCellTitle = ({
+    value,
+    onChange,
+    placeholder,
+    onUpdateTitle,
+    query,
+}) => {
+    const [loading, setLoading] = useState(false);
+
+    const generateTitle = useCallback(
```

Comment:
how does the frontend know if we have AI support or not?

Reply:
adding a .yaml config file for both frontend and backend.